circle.yml

```diff
@@ -12,4 +12,6 @@ dependencies:
 
 test:
   override:
+    - find . -not -path '*/.eggs/*' -not -path '*/integration-test*' -not -path '*/data/*' -name '*.py' | xargs flake8
+    - python setup.py test
```

Comment:
Do we also need a `python setup.py install` here to make the `vectordatasource` module available when `./scripts/setup_and_run_tests.sh` runs in the next line?